nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/pom.xml

```diff
@@ -117,6 +117,26 @@
             <version>1.19.0-SNAPSHOT</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk-translate</artifactId>
+            <version>1.12.328</version>
```

Comment:
These version references can be removed. The root Maven configuration includes the `aws-java-sdk-bom`, which controls the version for all AWS SDK Version 1 libraries.
```suggestion
```